azerothcore/azerothcore-wotlk
# DB restructuring - world - graveyard_zone

##### CHANGES PROPOSED:

-  Create TrinityCore like table structure for graveyard_zone
-  Read the graveyard zones from the table with the new structure
-  Modified .reload command to fit the new table
- Added all possible comments from TrinityCore branch


###### ISSUES ADDRESSED:


Close https://github.com/azerothcore/azerothcore-wotlk/issues/1189


##### TESTS PERFORMED:


Tested on Linux Ubuntu 18.10 with no build errors.

##### HOW TO TEST THE CHANGES:


- Make sure graveyards still work as intended. E.g. do some checks with dying and resurrecting.

Also:

In game there is the new command 
```
.reload graveyard_zone
```


##### KNOWN ISSUES AND TODO LIST:


- [ Keep up-to-date to TrinityCore if there are new changes ]


##### Target branch(es):

Master
